lib/regexp-examples/unicode_char_ranges.rb

```diff
@@ -0,0 +1,47 @@
+require 'pstore'
+
+module RegexpExamples
+  class UnicodeCharRanges
+
+    # These values were generated by: scripts/unicode_lister.rb
+    # Note: Only the first 128 results are listed, for performance.
+    # Also, some groups seem to have no matches (weird!)
+    STORE_FILENAME = 'unicode_ranges.pstore'
+
+    attr_reader :range_store, :store_location
+
+    def initialize(location="db/#{STORE_FILENAME}")
+      @store_location = location
+    end
+
+    def get(key)
+      range_store.transaction(true) do
+        ranges_to_unicode(range_store[key])
+      end
+    end
+
+    alias_method :[], :get
+
+    private
+
+    def ranges_to_unicode(ranges)
+      result = []
+      ranges.each do |range|
+        if range.is_a? Fixnum # Small hack to improve readability below
```

Comment:
Given that we're not using a pstore (which is not human readable), it might be worth removing this "hack" from the code? I only put it in, so that you don't end up with (in the old style) ugly definitions like:

    'blank' => ranges_to_unicode(9..9, 32..32, 160..160, .....)

What do you think?